api/services/rootfs/rootfs.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package containerd.v1;
+
+import "gogoproto/gogo.proto";
+import "github.com/docker/containerd/api/types/mount/mount.proto";
+import "github.com/docker/containerd/api/types/descriptor/descriptor.proto";
+
+service RootFS {
+	rpc Prepare(PrepareRequest) returns (PrepareResponse);
```

Comment:
After looking this over, I think we should call this `Unpack`. Then, `InitMounts` becomes prepare, which is exactly analogous to the `snapshotter.Prepare` method in that it creates an active layer.

I still think we need to flexible on what the base model is for this service. For example, when I "unpack", what do we get? Can they be listed?